parsec/core/fs/remote_loader.py

```diff
@@ -637,14 +639,17 @@ async def load_manifest(
             author = await self.remote_devices_manager.get_device(expected_author)
 
         try:
-            remote_manifest = BaseRemoteManifest.decrypt_verify_and_load(
-                rep["blob"],
-                key=workspace_entry.key,
-                author_verify_key=author.verify_key,
-                expected_author=expected_author,
-                expected_timestamp=expected_timestamp,
-                expected_version=expected_version,
-                expected_id=entry_id,
+            remote_manifest = cast(
```

Comment:
not needed with the "fake inheritance in .pyi" trick ^^

Reply:
This discussion will continue in https://github.com/Scille/parsec-cloud/pull/2838#discussion_r948964509 then